scapy/layers/inet.py

```diff
@@ -2076,6 +2075,7 @@ def fragleak2(target, timeout=0.4, onlyasc=0, count=None):
 class ICMPEcho_am(AnsweringMachine):
     """Responds to ICMP Echo-Requests (ping)"""
     function_name = "icmpechod"
+    send_function = staticmethod(sendp)
```

Comment:
Why did you change this?

Reply:
Yeah this has nothing to do with this PR, good catch